bootstrap.py

```diff
@@ -918,6 +928,46 @@ class MEOptions:
         migrate_systems(options.org, options.activationkey)
         if options.enablerepos:
             enable_repos()
+    elif options.new_capsule and options.foreman_fqdn:
+        # > ELIF new_capsule and foreman_fqdn set, will migrate to other capsule
+        #
+        # > will replace CA certificate, reinstall katello-agent, gofer
+        # > update system definition to point to new capsule for content,
+        # > Puppet, OpenSCAP and update Puppet configuration
+        # > MANUAL SIGNING OF CSR OR MANUALLY CREATING AUTO-SIGN RULE STILL REQUIRED!
+        # > API doesn't have a public provision for creating auto-sign entries yet!
+        options.force = True  # Need to set this to make get_bootstrap_rpm() to perform the right way
```

Comment:
can we change `get_bootstrap_rpm` to accept a `clean` Boolean, default it to `False` but pass `options.force` every time we call it instead. And then here don't set force to `true` but call `get_bootstrap_rpm(clean=True)`?

Just want to make sure that in future we don't actually want to omit something on `force=False` when moving capsules and can't or forget about it.